lib/generator._js

```diff
@@ -60,13 +60,14 @@ function ScriptGenerator(repositoryRoot, projectType, projectPath, solutionPath,
   projectType = projectType.toUpperCase();
 
   if (!scriptType) {
+    // use posh by default
     // If no script type is passed, use the default one
     if (projectType === ProjectType.wap || projectType === ProjectType.website || projectType === ProjectType.python) {
       // For .NET the default script type is batch
-      scriptType = ScriptType.batch;
+      scriptType = ScriptType.posh;
```

Comment:
This doesn't sound related to the PR. Also, the comment above needs adjustment, as it says default is batch.

I'm a bit worried about switching to posh for everyone, as I'm not sure it's been well tested.

Reply:
this is a side effect, I am pushing this to my repo for testing purpose, it will get reverted, and once I smash everything, it will be a NOOP

Reply:
Ah ok